package.json

```diff
@@ -69,7 +67,6 @@
     "gulp-replace": "^1.0.0",
```

Comment:
Still missing to remove old docs engine dependencies:
- `gulp-replace` & `@types/gulp-replace`
- `gulp-imagemin` & `@types/gulp-imagemin`
- `gulp-htmlmin` & `@types/gulp-htmlmin`
- `gulp-html-replace` & `@types/gulp-html-replace`